frontend/packages/ceph-storage-plugin/src/components/ocs-install/ocs-request-data.ts

```diff
@@ -52,7 +52,7 @@ export const getOCSRequestData = (
   } as K8sResourceKind;
 
   if (isMinimal) {
-    requestData.spec = {
+    requestData.spec = Object.assign(requestData.spec, {
```

Comment:
This can now be removed as the PR has merged.

Reply:
Yes, will rebase it.